### PR TITLE
add insert, modify species table, calc based species

### PR DESCRIPTION
**Pull request by smathermather**
* add insert statements to populate most of the lookup tables
* modify the species table (and all dependent views) so that we use the "official" updated species table maintained by Ohio EPA rather than the old table embedded in the spreadsheets
* calculations are based directly on species attributes in the table rather than codes derived from those attributes.  Also a few other minor changes that are pretty trivial.

### CODE REVIEW

File: backend/mapper/conf/vibi_views.sql [hunk 1]
```diff
@@ -238,56 +238,56 @@ CREATE MATERIALIZED VIEW reduced_fsd2_calculations_iv AS
...
 
-CREATE MATERIALIZED VIEW calculations_reduced_fsd1 AS
+CREATE MATERIALIZED VIEW calculations_reduced_fds1 AS
   SELECT a.plot_no,
-    sum(CASE WHEN c.code1 = 'carex' THEN 1.0 ELSE 0.0 END) AS carex,
-    sum(CASE WHEN c.code2 = 'cyper' THEN 1.0 ELSE 0.0 END) AS cyperaceae,
-    sum(CASE WHEN c.code1 = 'natDI' THEN 1.0 ELSE 0.0 END) AS dicot,
+    sum(CASE WHEN c.scientific_name SIMILAR TO '%Carex%' THEN 1.0 ELSE 0.0 END) AS carex,
+    sum(CASE WHEN c.family = 'Cyperaceae' THEN 1.0 ELSE 0.0 END) AS cyperaceae,
+    sum(CASE WHEN c.oh_status = 'native' AND c.type = 'DI' THEN 1.0 ELSE 0.0 END) AS dicot,
```
Comment (nmco) [inline backend/mapper/conf/vibi_views.sql]: oh_status and type columns don't exists in the species table:
https://github.com/cleveland-metroparks/vibi/blob/master/vibi_tables.sql#L244-L245

File: backend/mapper/conf/vibi_views.sql [hunk 2]
```diff
@@ -238,56 +238,56 @@ CREATE MATERIALIZED VIEW reduced_fsd2_calculations_iv AS
...
     sum(CASE WHEN c.shade = 'shade' OR c.shade = 'partial' THEN 1.0 ELSE 0.0 END) AS shade,
-    sum(CASE WHEN c.code2 = 'natwtldSH' THEN 1.0 ELSE 0.0 END) AS shrub,
-    sum(CASE WHEN c.hydro = 'hydrophyte' THEN 1.0 ELSE 0.0 END) AS hydrophyte,
-    sum(CASE WHEN c.groupp = 'SVP' THEN 1 ELSE 0.0 END) AS svp,
+    sum(CASE WHEN c.oh_status = 'native' AND c.ncne = 'FACW' OR c.ncne = 'OBL' AND c.form = 'shrub' THEN 1.0 ELSE 0.0 END) AS shrub,
```
Comment (nmco) [inline backend/mapper/conf/vibi_views.sql]: oh_status was named nativity int the species table definition:
https://github.com/cleveland-metroparks/vibi/blob/master/vibi_tables.sql#L245

ncne column was commented in the species table:
https://github.com/cleveland-metroparks/vibi/blob/master/vibi_tables.sql#L248

File: backend/mapper/conf/vibi_views.sql [hunk 3]
```diff
@@ -238,56 +238,56 @@ CREATE MATERIALIZED VIEW reduced_fsd2_calculations_iv AS
...
+    sum(CASE WHEN c.ncne = 'FACW' OR c.ncne = 'OBL' THEN 1.0 ELSE 0.0 END) AS hydrophyte,
```
Comment (nmco) [inline backend/mapper/conf/vibi_views.sql]: ncne column was commented in the species table:
https://github.com/cleveland-metroparks/vibi/blob/master/vibi_tables.sql#L248

File: backend/mapper/conf/vibi_views.sql [hunk 4]
```diff
@@ -238,56 +238,56 @@ CREATE MATERIALIZED VIEW reduced_fsd2_calculations_iv AS
...
+    sum(CASE WHEN c.type = 'SVP' THEN 1 ELSE 0.0 END) AS svp,
```
Comment (nmco) [inline backend/mapper/conf/vibi_views.sql]: type column was commented in the species table:
https://github.com/cleveland-metroparks/vibi/blob/master/vibi_tables.sql#L244

**Comment by nmco** [comment]
Regarding this two points:
- add insert statements to populate most of the lookup tables
- modify the species table (and all dependent views) so that we use the "official" updated species table maintained by Ohio EPA rather than the old table embedded in the spreadsheets
where are this changes ?

**Comment by smathermather** [comment]
Hmm, good question. I was blindly pushing from John's changes. I'll ping him and look into it myself too if I get a chance tomorrow.

**Pull request merged**